composer/loggers/neptune_logger.py

```diff
@@ -58,7 +60,7 @@ def __init__(
         project: Optional[str] = None,
         api_token: Optional[str] = None,
         rank_zero_only: bool = True,
-        upload_artifacts: bool = False,
+        upload_checkpoints: bool = False,
```

Comment:
Removing `upload_artifacts` would make this a breaking change.
Lets support both `upload_artifacts` and `upload_checkpoint`, and add a deprecation notice to `upload_artifacts`.

Reply:
It's still under `1.0.0` version, so I guess breaking changes like that might still be allowed

Reply:
@SiddhantSadangi done. @normandy7 would you like to review the warning message?